gabrielaEnache/src/main/java/oop/store/AbstractOrder.java

```diff
@@ -0,0 +1,106 @@
+package oop.store;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractOrder implements Order {
+    protected final long id;
```

Comment:
protected means that the attribute can be manipulated within the child class, however you have defined some getters and setters and these variables can be declared as private.

Reply:
By setting id private, we won't be able to inherit it in sub classes. 
However by declaring it **protected final** and initialize it in constructor the attribute can't be manipulated within the child class.

Reply:
please read my message from teams, and try once again to change the access modifier, let me know if there are some problems

Reply:
Fixed